net-misc/zerotier/zerotier-1.2.8.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic systemd toolchain-funcs
+
+HOMEPAGE="https://www.zerotier.com/"
+DESCRIPTION="A software-based managed Ethernet switch for planet Earth"
+SRC_URI="https://github.com/zerotier/ZeroTierOne/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+S="${WORKDIR}/ZeroTierOne-${PV}"
+
+RDEPEND="
+	dev-libs/json-glib
+	net-libs/http-parser
```

Comment:
This and the two below need `:=` since they use subslots to express SONAME.

Reply:
All of them?

```
koel? ( dev-libs/json-glib:= )
```
https://github.com/gentoo/gentoo/blob/master/media-sound/pragha/pragha-1.3.92.ebuild

```
net-libs/http-parser:=
```

https://github.com/gentoo/gentoo/blob/master/dev-libs/libgit2/libgit2-0.27.0.ebuild

```
nat-pmp? ( net-libs/libnatpmp:= )
net-libs/miniupnpc:=
```
https://github.com/gentoo/gentoo/blob/master/net-p2p/airdcpp-webclient/airdcpp-webclient-2.3.0.ebuild

Reply:
I don't think it's strictly useful for json-glib but I suppose it wouldn't hurt either.